### DISCUSSION TITLE
Attention à la charge des véhicules électriques à domicile - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#39] [Comments]
# Attention à la charge des véhicules électriques à domicile - Haade.fr

Retour d’expérience sur les risques d’incendies des recharges des véhicules électriques à domicile et surveiller le problème grâce à la domotique.

https://haade.fr/fr/blog/attention-charge-vehicule-electrique-a-domicile

**Comment by bonac** [comment]
Bonjour,
je voudrais nuancer la revue dithyrambique ci dessus:
  - vous comparez un DJ Legrand qui a 5 ans d'utilisation avec un DJ tout neuf. On en reparlera dans 5 ans en conditions identiques (courant de 40A et non 30 ...)
  - pour avoir démonté beaucoup d'équipements chinois, il est classique de trouver beaucoup d'erreurs de conception, voire d'arnaques claires et nettes (des contacts trop petits, des parafoudres 63 ou 100A avec des composants minuscules et sans même le ressort qui indique que la cartouche a grillé, des fils trop petits, des câbles avec prise de terre SANS le fil de terre, la liste est très très longue). Oui beaucoup de fabriquants non chinois font fabriquer en Chine MAIS ils font des contrôles de qualité réguliers.
  - le test devrait aussi montrer le démontage des deux pour comparer la taille des contacts et conducteurs.
  - avoir tous les certificats ecrits sur le boitier n'est en aucun cas un gage de qualité, surtout venant de la part d'un pays qui a cloné le symbole CE Européen pour en faire le même symbole mais qui signifie "China Export".
  - le DJ que vous proposez ne coupe PAS le neutre contrairement à celui de Legrand (le danger du "retour de neutre", bien connu des électriciens)
  - perso je préfère Schneider à Legrand, mais comme toujours tout dépend des séries
  - même si ça existe, c'est une abérration de vouloir couper du 40A sous un format DIN une largeur 18mm, il n'y a pas assez de place, il faut passer sur du module double pour que les contacts soient assez gros. D'ailleurs les contacts de vissage des modules doubles sont bien plus gros, participant aussi à la dissipation thermique (qui passe aussi par les conducteurs il faut le savoir). 40A mériterait du câble plus gros qu'un simple peigne 10mm2, par sécu dans le tableau
  - il ne fait pas oublier qu'un boitier électrique doit "respirer" et donc être bien ventilé. Le format rail DIN et petits modules laisse trop facilement enfermer les DJ dans des emplacements quasi étanches et riquiquis.
  - mais surtout le plus important: le DJ proposé puisqu'il est WiFi, comporte beaucoup d'électronique, et compte tenu de la chaleur, il va se passer ce qui arrive de plus en plus de nos jours: les composants électroniques vont vieillir vite et un condensateur chimique ou un transistor va claquer, avec un mini début d'incendie. C'est le pb qu'on constate assez souvent sur des ampoules LEDs au plafond. Sans parler de toute la place que prend cette électronique, au détriment des contacts et conducteurs.
  - donc il serait plus sûr que le DJ proposé pilote un vrai relais de puissance comme le https://www.se.com/fr/fr/product/A9C20842/acti9-ict-contacteur-auxiliarisable-2p-40a-2no-230-240vca-50hz/ certes du coup plus de contrôle de sûr-courant, température, etc.
  - au final un vrai DJ diff double comme celui-ci https://www.se.com/africa/fr/product/A9R41240/residual-current-circuit-breaker-rccb-acti9-iid-2p-40a-ac-type-30ma/ permettrait de tout remplacer. Le monitoring de température et courant pouvant être fait avec de Raspberry Pi et capteur de température et bobines transfo de courant, pour centraliser plus de choses.
 Merci pour vos revues que je découvre.

**Reply by haade-administrator** [reply]
Salut tu n'as pas tort, et je dois dire que ta réponse est constructive et très pertinente, pour le prix d'achat des Legrand il est anormal qu'il monte autant en température, quant au module chinois il est juste là pour couper le courant en cas de trop forte chaleurs. Ce qui est dommage c'est le fait de ne pas pouvoir automatiser la wallbox pour lui donner en consigne de baisser en ampérage en cas de températures trop élevés.

**Reply by bonac** [reply]
Oui, Legrand se gave surtout sur ces séries "Résidentielles" (rési), mais 32A sur différentiel THERMIQUE en 1 module de large, collé à l'autre qui lui aussi se prend les 32A, ça ne m'étonne pas que ça chauffe.
La wallbox ne va pas se casser la tête: elle estime que tout va bien en amont ... Quelle mesure de temp pourait-elle faire: elle-même, l'armoire électrique ? On va peut-être finir avec des ventilateurs dans des armoire électriques. je connais une grosse armoire d'un double appart, planquée dans un placard où il fait 35 degrés dedans (je n'imagine pas combien dans les modules), avec en plus du matos informatique au dessus car seule place possible. Pas bon du tout :-(((
Le soucis c'est que couper à distance par WiFi si il fait trop chaud, ça va créer des étincelles d'extra-ruptures, donc les contacts vont s'esquinter à la longue et charbonner, et donc chauffer :-( Il faudrait vraiment pouvoir couper ou moduler sur le chargeur lui-même à distance.

**Comment by Michoumbk** [comment]
Bonjour
Je viens d'acheter cette article sur ALIEXPRESS. Je voudrai me servir de tuya pour le piloter, dans l'appli il n'y a rien pour régler l'amperage, c'est pareil pour vous ? Faut t'il faire quelque chose de spécifique pour afficher cette partie ?
Merci pour votre réponse
michoumbk

**Reply by haade-administrator** [reply]
Salut je n'utilise pas tuya mais z2m via home assistant, as-tu installé l'application smart life

**Comment by Michoumbk** [comment]
Oui et j'ai pas le réglage des amperes
Cdlm

**Comment by erwan56450** [comment]
Pour ma part, je recommande un interrupteur différentiel Schneider de 63 A (même si c'est plus cher), ainsi qu'un disjoncteur bipolaire de 40 A. En effet, un disjoncteur de plus forte capacité dissipe mieux la chaleur, et un pouvoir de coupure de 10 kA, comme celui-ci, est nettement supérieur aux 6 kA habituels, ce qui en dit long sur sa robustesse.
https://www.123elec.com/schneider-acti9-ic60n-disjoncteur-40a-2p-courbe-c-10ka-230v-a9f77240.html

Il est également important de ne pas coller les modules afin de laisser de l'espace pour la ventilation.

Pour ceux qui souhaitent piloter ou automatiser la charge, utiliser un interrupteur comme le Tongou avec 7000 W est risqué, car il pourrait surchauffer. Il est préférable de piloter directement le chargeur, qui enverra un signal à la voiture pour stopper la charge. Cela coûte environ 150 euros sur des sites chinois :

https://fr.aliexpress.com/item/1005005504929776.html?spm=a2g0o.order_list.order_list_main.123.29335e5bBE99kO&gatewayAdapt=glo2fra

https://youtu.be/NQe61K7PITc pilotable sur Home Assistant.

**Reply by haade-administrator** [reply]
C'est une manière de voir les choses et tu as raison sur le fait de passer sur un disjoncteur 63A, dans mon cas la wallbox est difficilement domotisable, sinon j'aurai favorisé un contrôle direct

**Comment by benppppp** [comment]
Bonjour,
Est-ce que c'est bien le peigne qui a chauffé comme je crois le voir sur la photo?
Le différentiel legrand utilisé a t il la référence 411638?
Si oui, d'après sa notice, il ne faut pas le raccorder directement au disjoncteur 40A par un peigne raccourci (comme visible sur la photo du différentiel brulé)). Il faut ajouter deux éléments d'espacement ref 406307 entre le différentiel et et le disjoncteur pour augmenter la masse du peigne et ainsi de limiter son échauffement.
A noter: cette info est visible uniquement sur le site pro Legrand et pas sur le site particulier. Je ne sais pas non plus quand cette info a été publiée dans la notice, peut-être après la pose du différentiel...
Lien vers la notice:
https://assets.legrand.com/pim/NP-FT-GT/LE14820AA.pdf
Lien vers le différentiel Legrand:
https://www.legrand.fr/pro/catalogue/interrupteur-differentiel-dx3-id-arrivee-haut-a-vis-et-depart-haut-automatique-2p-230v-40a-typea-30ma-2-modules
Lien vers l’élément d'espacement Legrand:
https://www.legrand.fr/pro/catalogue/element-despacement-passe-fil-05-module

**Reply by haade-administrator** [reply]
merci pour toutes ces infos en or, je vais probablement mettre à jour l'article avec ces connaissances

**Reply by bonac** [reply]
Si le peigne chauffe c'est qu'il est trop petit. Ce qui ne m'étonne pas, j'ai toujours été circonspect sur ces tailles en forts courants.
Autant le doubler ou mettre des fils à la place de section plus importante.
Si ce sont les CONTACTS des DJ qui chauffent et se servent du rail pour refroidir là c'est une autre paire de manche, on en revient au pb de DJ de forts courants en un seule largeur DIN = contacts trop petits.
C'est moche que Legrand ne mette pas cet avertissement sur les notices non  pros. Mais plus rien ne m'étonne ...

**Comment by fabien4444** [comment]
Bonjour à tous,
article très intéressant qui me fait douter de mon installation. J'aimerais vos avis d'expert svp.

j'ai préparé l'arrivée de ma VE cette année avec l'installation d'une bonne de recharge 7.4kw avec un dispositif de ligne dédiée en 3G10 de 16m de long et protégé dans le tableau avec le couple 
- Disjoncteur Hager MJT740 40A
- Interrupteur différentiel Hager BDH240F 40A 30mA A HI

Le tableau électrique se situe dans une pièce chauffée?

Que pouvez dire sur la sécurité de cet ensemble? Merci par avance

Voici une photo:
https://i.ibb.co/0q25S6Y/IMG-20250112-101200255-HDR.jpg

**Reply by bonac** [reply]
Il faut vérifier que les câbles/peignes entre le DJ et le diff. sont assez gros, pour aider au refroidissement des contacts du DJ.
Accessoirement cf la précédente remarque, ça reste un DJ en MONO-largeur DIN, donc des petits contacts. Il convient de surveiller la chauffe après une heure de fonctionnement par exemple.
Eloigner le diff. du DJ avec des câbles plus long c'est tout bénèf en terme de refroidissement ! (2ème côté libre pour le DJ pour se refroidir et fils plus longs qui aussi refroidissent mieux)
Le pb plus haut était la commande à distance, donc RÉGULIÈRE du DJ. Si le DJ n'est là qu'en sécu, ou en coupure pour maintenance et qu'il ne chauffe pas, pas de soucis. Si il est amené régulièrement à couper, en PLEINE CHARGE, là c'est un vrai pb, car avec le "charbonnage" et la destruction petit à petit des contacts par les étincelles d'extra rupture, les contacts vont chauffer de plus en plus. Ca n'est pas pour rien que les gros contacteurs/relais sont souvent montés avec les contacts en série (et non en parallèle), pour limiter les tensions et donc les destructions des contacts.
Vous c'est juste en sécu, vu la photo, donc çe n'est pas votre cas. Faut juste vérifier que ça na chauffe pas.
Le DIN c'est bien et pratique mais c'est courant qu'ils fasse 35 degrés dans le coffret, sans parler de l'été. Pas bon du coup pour les équipement informatiques si tout est au même endroit...

**Reply by bonac** [reply]
Il suffit de regarder la taille d'un DJ d'abonné certes 60A (juste le double), par rapport à un module DIN pour se dire que côté chaleur on n'est pas à armes égales.
https://www.123elec.com/schneider-resi9-db60-disjoncteur-d-abonne-monophase-15-30-45a-500ma-differentiel-instantane-r9ft645.html
Et encore même les DJ d'abonnés finissent par cramer à la longue ...
Et les anciens DJ noirs étaient encore plus gros ;-)

**Reply by benppppp** [reply]
@fabien4444 Hager MJT740+BDH240F ça chauffe mais j'en ai pas encore vu de brulé sur les forum. Voir ici pour le relevé de température:
https://www.blogtesla.fr/forum/viewtopic.php?p=368605#p368605
A comparer avec la version Legrand à peigne 2 dents ici:
https://www.forumconstruire.com/construire/topic-468843-disjoncteur-c40a-chauffe.php
Difficile de conclure car la mesure Hager est beaucoup moins bien faite que la mesure Legrand

@bonac pour les pros, aucun des installateurs agréés IRVE que j'ai contacté n'était au courant que les peignes 2 dents étaient interdits par Legrand. Les fabricants n'osent pas communiquer et en attendant les disjoncteurs brulent...

**Reply by bonac** [reply]
AFFLIGEANT mais très instructif les liens, merci :-)
15W de dissipation pour un DJ et c'est considéré comme "_normal_", oui enfin c'est juste un petit fer à souder qui peut monter à 350 degrés en ponctuel !!!
8mm2 de section pour les dents des peignes et on s'étonne que ça chauffe et dissipe mal ... ;-)
La photo de l'additif diff BDF240F sur blogtesla fait peur: les conducteurs d'origine sont bien petits...
Ben si ya un DJ brulé sur un forum, il ya a malheureusement des chances que l'utilisateur ait d'autres choses à règler ensuite que poster sur un forum.
Je pense que je vais investir dans une caméra thermique, trop de soupçons et plus du tout confiance dans les constructeurs de marque, alors les autres ... On me dit souvent que je mets des câbles trop gros et au dessus de la norme, je ne le regrette plus du tout ;-)

**Comment by Rexxregis** [comment]
Bonjour,

Expérience qui aurait pu très mal se terminer...

Un point important à relever tout de même. Même si l'installation par un non professionnel d'une borne de recharge est techniquement possible, un décret du 12 janvier 2017 impose aux particuliers de faire appel à un électricien certifié IRVE pour l’installation d’une wallbox de puissance supérieure à 3,7 kW. Si votre borne délivre 7,2 kW et à environ 5 ans, elle aurait dû être installée par un professionnel... Peut être pas essentiel d'un point de vue technique, mais en cas de sinistre je ne suis pas certain qu'une compagnie d'assurance couvrirait les dégâts occasionnés...

Comme mentionné par benppppp, les notices actuelles (professionnelles mais aussi grand public) des interrupteurs différentiels Legrand interdisent le montage pratiqué si le disjoncteur associé à l'interrupteur différentiel à une puissance supérieure ou égale à 25kW. En revanche, seules les notices professionnelles des disjoncteurs reprennent cette restriction d'installation. La mention de cette restriction est bien absente de la notice du disjoncteur 32A grand public (le 40A est exclusivement professionnel et le 25A n'existe pas en version bornes auto).

Le montage proposé par Legrand pour ses propres bornes de recharge utilise un disjoncteur différentiel 40A type F (sans protection intégrée) ou un interrupteur différentiel dans la borne associé à un disjoncteur dans le tableau électrique (protection intégrée). Donc aucun problème de distance entre interrupteur différentiel et disjoncteur.  Un disjoncteur différentiel est certes plus cher mais totalement insignifiant comparé  à un sinistre. Il reste que le disjoncteur différentiel ne doit pas être "collé" à un autre appareil similaire pour permettre la dispersion de la chaleur...

Il est toujours possible de contacter le service consommateur Legrand (même pro!) pour obtenir des précisions. Il est toujours très réactif et extrêmement professionnel. Il est probable qu'il aurait été intéressé par votre accident (probablement pas un cas unique).

Cordialement
